README.md

```diff
@@ -102,6 +102,24 @@ Your project will be compiled ahead of time (AOT), and then the resulting bundle
 - UMD modules result in code that cannot be properly optimized. For best results, prefer ES6 modules whenever possible. This includes third-party dependencies: if one is published in both UMD and ES6 modules, go with the ES6 modules version.
 - During a production build, CommonJs modules will be automatically converted to ES6 modules. This means you can use them and/or require dependencies that use them without any issues.
 
+# Internationalization
+
+Put `i18n` attribute to your html tag to mark it for translation, more information here: https://angular.io/docs/ts/latest/cookbook/i18n.html
+
+## Create a translation source file
+
+```bash
+# Your translation file will be generated here `dist/locale`
+$ npm run i18n
+```
+
+## Production build with your language
+
+```bash
+# Build prod app with the language file `dist/locale/messages.en.xlf`
```

Comment:
Did you mean we divide the translation task to 2 process:
First, run `npm run i18n` to generate `dist/locale/messages.xlf` file.
(Developer will translate this file and put it into `dist/e2e` folder)
Second, run `npm run i18n.merge` (for example) to merge to existing translations in `src/assests/locale`